appinventor/appengine/src/com/google/appinventor/client/DesignToolbar.java

```diff
@@ -302,6 +302,11 @@ public void execute() {
     }
   }
 
+ public void switchToBlocksEditor() {
```

Comment:
I don't know. I didn't write it.
Looks like it's being used in BlocklyPanel and DesignToolbar, and is used in exportMethodsToJavascript(). They wrote one for SwitchToFormEditor, too.